.github/workflows/test-template.yml

```diff
@@ -165,7 +166,7 @@ jobs:
           skip-tsc: false
 
       # Run the functional test suite
-      - run: pnpm run test:functional --silent --shard ${{ matrix.shard }} --engine-type ${{ matrix.queryEngine }}
+      - run: pnpm run test:functional --silent --shard ${{ matrix.shard }} --engine-type ${{ matrix.queryEngine }} --preview-features "${{ matrix.previewFeatures }}"
```

Comment:
Unrelated: It would be nice to have this kind of testing for all preview features at once.

Reply:
(And not required now)